swap_meet/clothing.py

```diff
@@ -0,0 +1,9 @@
+from .item import Item
+
+
+class Clothing(Item):
+    def __init__(self, category="", condition=0, condition_description=" "):
+        super().__init__("Clothing", condition)
```

Comment:
we know that class Clothing's category will always be "clothing." So, we can remove the parameter altogether. I'm not sure what the `condition_description` attribute is being used for, but I think we can get rid of it since it is never assigned or called.

```suggestion
    def __init__(self, condition=0):
        super().__init__("Clothing", condition)
```